CHANGES.md

```diff
@@ -63,6 +63,7 @@
 * Beam modules that depend on Hadoop are now tested for compatibility with Hadoop 3 ([BEAM-8569](https://issues.apache.org/jira/browse/BEAM-8569)). (Hive/HCatalog pending)
 * Publishing Java 11 SDK container images now supported as part of Apache Beam release process. ([BEAM-8106](https://issues.apache.org/jira/browse/BEAM-8106))
 * Added Cloud Bigtable Provider extension to Beam SQL ([BEAM-11173](https://issues.apache.org/jira/browse/BEAM-11173), [BEAM-11373](https://issues.apache.org/jira/browse/BEAM-11373))
+* Added a schema provider for thrift data ([BEAM-11338](https://issues.apache.org/jira/browse/BEAM-11338))
```

Comment:
It could be nice if this linked to the (future) javadoc for ThriftSchema. The javadoc doesn't exist now, but it will once this file is turned into release notes and a blog post.

Reply:
I added comprehensive javadoc for `ThriftSchema`, so the future is almost here :). Please have a look and let me know if I covered everything.
By linking to the future javadoc, you mean pointing to this URL: https://beam.apache.org/releases/javadoc/2.27.0/org/apache/beam/sdk/io/thrift/ThriftSchema.html ?
Everything else in the `CHANGES.md` file seems to point to Jira, but I'm fine either way.

Reply:
Yeah you're right we don't do that for anything else, this is fine.

The javadoc looks great! My only nit: All the bullet points should be capitalized and have periods

Reply:
Thanks, done.